### ISSUE TITLE
GRPO实现可能有误

### ISSUE THREAD

**Issue by macheng6** [OpenRLHF/OpenRLHF#768]
reward中已经计算了kl loss，因此在advantages也包含了kl loss，在开启use_kl_loss参数的情况下，最终的ppo_loss又会再次计算一次kl loss，这应该与原始论文不符。

**Comment by KAKSIS** [comment]
compute reward中虽然计算了kl，但是前边make_experience中如果启用了 args.use_kl_loss，kl会被置0，所以应该还是只有算loss的时候一次kl
应该是这里置0的
##RemoteExperienceMaker.make_experience
kl = torch.zeros_like(action_log_probs, dtype=action_log_probs.dtype, device=device)

**Comment by macheng6** [comment]
哦哦，确实是的。感谢
